OsmAnd/src/net/osmand/plus/activities/SavingTrackHelper.java

```diff
@@ -88,77 +90,81 @@ public SavingTrackHelper(OsmandApplication ctx){
 		prepareCurrentTrackForRecording();
 
 		updateScript = "INSERT INTO " + TRACK_NAME + " (" + TRACK_COL_LAT + ", " + TRACK_COL_LON + ", "
-				+ TRACK_COL_ALTITUDE + ", " + TRACK_COL_SPEED + ", " + TRACK_COL_HDOP + ", " 
-				+ TRACK_COL_DATE  + ", " + TRACK_COL_HEADING + ")"
+				+ TRACK_COL_ALTITUDE + ", " + TRACK_COL_SPEED + ", " + TRACK_COL_HDOP + ", "
+				+ TRACK_COL_DATE + ", " + TRACK_COL_HEADING + ")"
 				+ " VALUES (?, ?, ?, ?, ?, ?, ?)"; //$NON-NLS-1$ //$NON-NLS-2$
 
-		insertPointsScript = "INSERT INTO " + POINT_NAME + " VALUES (?, ?, ?, ?, ?, ?, ?)"; //$NON-NLS-1$ //$NON-NLS-2$
+		insertPointsScript = "INSERT INTO " + POINT_NAME + " VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)"; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
add column names to "insert into ... (...) values ()"  similar to updateScript, so we know in which order columns are inserted! better always specify